### PR TITLE
docs: adds google calendar bot docs

### PR DESCRIPTION
**Pull request by muromeo1**
## PR Description

This PR adds a detailed documentation about the `Google Calendar` action to the bot.

[[Preview]](https://deploy-preview-54--callbell-docs.netlify.app/bot/editor/actions/logic/google_calendar/)

### CODE REVIEW

File: docs/bot/editor/actions/logic/google_calendar.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+**Event starts at**: The time when the event will start. The required time format is: `HH:MM`.
+
+**Event duration (in minutes)**: The length of the event in minutes.
+
+**Send event updates to**: The recipients who will receive updates about the event. The options are: `None` or `All`.
+
+**Create meet link**: Option to generate a link for a virtual meeting associated with the event.
+
+:::tip
+You can use variables to dynamically populate these details
```
Comment (camilamiz) [inline docs/bot/editor/actions/logic/google_calendar.md]:
```suggestion
You can use variables to dynamically populate these fields
```

File: docs/bot/editor/actions/logic/google_calendar.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+<img src={googleCalendarVariables} width={500} />
+
+2. `Summary`, `month` and `duration` were set as fixed values.
+
+3. We used a combination of actions to dynamically retrieve the day:
+
+- [Send message](/bot/editor/actions/interaction/send_message) to send the message asking about which day is the best.
+- [Wait user's answer](/bot/editor/actions/interaction/wait_user_answer) to get the answer.
+- [Set variable](/bot/editor/actions/logic/set_variable) to save the result from the answer in our variable `day`.
```
Comment (camilamiz) [inline docs/bot/editor/actions/logic/google_calendar.md:110]: Maybe it would be cool to add an image showing how the combination will look like. WDYT?

**Review by camilamiz — APPROVED** [review]
LGTM!

File: docs/bot/editor/actions/logic/google_calendar.md (hunk 2 above)
Comment (muromeo1) [reply docs/bot/editor/actions/logic/google_calendar.md:110]: Good point

**Pull request merged**